code/game/objects/structures/loot_piles.dm

```diff
@@ -522,3 +522,51 @@ Loot piles can be depleted, if loot_depleted is turned on.  Note that players wh
 		/obj/item/clothing/suit/armor/alien/tank,
 		/obj/item/clothing/head/helmet/alien/tank,
 	)
+
+/obj/structure/loot_pile/surface/bones
+    name = "bone pile"
+    desc = "A pile of various dusty bones. Your graverobbing instincts tell you there might be valuables here."
+    icon = 'icons/obj/bones.dmi'
+    icon_state = "bonepile"
+    delete_on_depletion = TRUE
+
+    common_loot = list(
+        /obj/item/weapon/bone,
+        /obj/item/weapon/bone/skull,
+        /obj/item/weapon/bone/skull/tajaran,
+        /obj/item/weapon/bone/skull/unathi,
+        /obj/item/weapon/bone/skull/unknown,
+        /obj/item/weapon/bone/leg,
+        /obj/item/weapon/bone/arm,
+        /obj/item/weapon/bone/ribs,
+        /obj/item/weapon/arrow
```

Comment:
Not really sure why this is in this list? That item, as far as I can tell, only exists to be thrown by the crossbow, and even then I'm not sure it works for that.

Reply:
My mindset was along the lines of "This is something that might've killed the person all these bones belong to." I was also thinking of putting a bullet casing in there as well or something but it slipped my mind. I could remove it if you feel it's too out of place.

Reply:
It does work for the crossbow, by the by. I did test that, and it works.

Reply:
Huh. Good to know. I'd still remove this. There are knives in the uncommon loot, that's... probably fine?
And a lot of what'll be killing people, ICly, is the climate.

Reply:
Fair enough. Can do.

Reply:
Thank you.